component/http/middleware.go

```diff
@@ -131,6 +132,21 @@ func NewLoggingTracingMiddleware(path string) MiddlewareFunc {
 	}
 }
 
+// NewRateLimitingMiddleWare creates a MiddlewareFunc that adds a rate limit to a route.
+func NewRateLimitingMiddleWare(limiter *rate.Limiter) MiddlewareFunc {
```

Comment:
Small nitpick, but all other functions are named like `Middleware` instead of `MiddleWare`. 

We could also add the function signature on the relevant [documentation page](https://github.com/beatlabs/patron/blob/master/docs/components/HTTP.md#helper-middlewares), along the other middlewares!

Reply:
Sure, let me add these.

Reply:
Fixed the function name, will add relevant documentation once I have clarity on burst